Marlin/SanityCheck.h

```diff
@@ -153,6 +153,8 @@
      */
     #ifdef AUTO_BED_LEVELING_GRID
 
+#ifndef DELTA_PROBABLE_RADIUS
```

Comment:
If you add lines like this, feel free to include them in the cascade, indenting them to the current level, and the lines between one level more.